### PR TITLE
config: add new dbt download script

### PR DESCRIPTION
**Pull request by michaelefisher**
Before you run the script, you'll need to auth to the dev Airbyte Kubernetes cluster:
```shell
gcloud config set CTA_PROJECT_ID_FOR_DEV
gcloud container clusters get-credentials airbyte-dev-cluster --region us-central1
```

Full instructions are here: https://github.com/community-tech-alliance/airbyte-kubernetes-infrastructure#kubernetes-setup

Once you choose the "Copy dbt from an Airbyte Workspace" from `./cta_dbt_helper.sh`, go to ensure you've chosen "Normalized tabular data" under the "Transformation" tab. Then, sync!

Screenshot 2023-10-23 at 16 17 02

You'll see output noting that the script is looking for a normalization pod. Once it finds it, your dbt should download to your local machine.

You'll see something like this:
```shell
Waiting for container main in pod normalization-normalize-3636-0-uutfg to be running... Current status: {"terminated":{"containerID":"containerd://1f433826ef4d231467c282aae75e032b27ddb2276d9cabf82a64e8b820bbb82b","exitCode":0,"finishedAt":"2023-10-23T23:58:13Z","reason":"Completed","startedAt":"2023-10-23T23:57:30Z"}}
Waiting for container main in pod normalization-normalize-3637-0-ksqtt to be running... Current status: {"terminated":{"containerID":"containerd://2359f11ac544d23e10555d21dd2d69ca94613a44929f738567d97e955b4801a7","exitCode":0,"finishedAt":"2023-10-24T00:07:04Z","reason":"Completed","startedAt":"2023-10-24T00:06:33Z"}}
Container main in pod normalization-normalize-3638-0-oajxs is now running.
Copying /config directory from main in pod normalization-normalize-3638-0-oajxs to local machine...
tar: Removing leading `/' from member names
Successfully copied /config directory.
Airbyte normalization exported to local directory -> airbyte_dbt_export/
```

Made a few changes elsewhere, but tested and everything works end-to-end.

If you run the sync multiple times, you'll notice that the script finds terminated pods. That's fine: it'll start downloading dbt again once it finds a new running pod.

### CODE REVIEW

**Comment by michaelefisher** [comment]
Question: does this need to work in prod *and* dev? If so, that's fine - just need to update instructions.

**Comment by kanelouise** [comment]
> Question: does this need to work in prod _and_ dev? If so, that's fine - just need to update instructions.

I'd say it'd be good for this to work in both dev and prod. I think folks may run this by default in dev, but there may be cases where prod is the best way to run it.

File: utils/cta_dbt_helper.sh [hunk 1]
```diff
@@ -9,41 +9,19 @@ copy_dbt_from_airbyte() {
...
-        fi  
-        chmod +x copy_airbyte_workspace.sh
-        cd $ROOT_PATH
-    fi
-
     # Run script to copy Airbyte Workspace to local
     echo "Starting job to copy Airbyte workspace.."
-    ./.cta/copy_airbyte_workspace.sh -e $ENVIRONMENT -w $WORKSPACE_ID
+    # ./copy_dbt_from_k8s_pod.sh <POD_NAME_REGEX_PATTERN> <NAMESPACE> <CONTAINER_NAME>"
+    ./copy_dbt_from_k8s_pod.sh "normalization" "airbyte" "main"
```
Comment (jitoquinto) [inline utils/cta_dbt_helper.sh:16]: Can these paths be prepended with $ROOT_PATH. Want this script to not care which directory it is being ran from. 

```shell
./$ROOT_PATH/utils/copy_dbt_from_k8s_pod.sh "normalization" "airbyte" "main" 
mkdir -p $ROOT_PATH/.cta/airbyte_dbt_export/
mv config/* $ROOT_PATH/.cta/airbyte_dbt_export/
```

**Review by jitoquinto — COMMENTED** [review]
Yep, agree with Kane that it would be good to run in both envs. Left a comment regarding paths but other than that lgtm!

**Comment by kanelouise** [comment]
I'm curious what sync you used to test this, @michaelefisher. I have gotten stuck while running it on the Impactive sync. Here are the  errors:
`sh utils/cta_dbt_helper.sh
Starting job to copy Airbyte workspace..
utils/cta_dbt_helper.sh: line 16: ./copy_dbt_from_k8s_pod.sh: No such file or directory
mv: rename config/* to .cta/airbyte_dbt_export/*: No such file or directory
rm: config/: No such file or directory
Failed to grab normalization dbt from -e dev Airbyte`

**Comment by michaelefisher** [comment]
This was addressed in a downstream PR.

**Review by kanelouise — APPROVED** [review]

**Pull request merged**